packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -341,6 +391,29 @@ class IconButton extends StatelessWidget {
   /// Null by default.
   final ButtonStyle? style;
 
+  /// The optional selection state of the icon button.
+  ///
+  /// If this property is null, the button will be a non-toggle button; otherwise,
+  /// the button will be a toggle button with the corresponding selection state.
+  ///
+  /// This property is only used for Material 3 [IconButton].
+  final bool? isSelected;
+
+  /// The optional icon to display inside the button when [isSelected] is true.
+  ///
+  /// The [Icon.size] and [Icon.color] of the icon is configured automatically
+  /// based on the [iconSize] and [color] properties of _this_ widget using an
+  /// [IconTheme] and therefore should not be explicitly given in the icon
+  /// widget.
+  ///
+  /// This property can be null. The original icon will be used for both selected
+  /// and unselected status if it is null.
+  ///
+  /// This property is only used for Material 3 [IconButton].
+  ///
+  /// See [Icon], [ImageIcon].
```

Comment:
We typically give a description for these "See also" entries. Take a look at some of the other widgets for examples.